packages/wagmi/src/client.ts

```diff
@@ -183,9 +187,16 @@ export class Web3Modal extends Web3ModalScaffold {
       disconnect
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
+    let siweControllerClient: SIWEControllerClient | undefined = undefined
+    if (siweConfig) {
```

Comment:
`new Web3ModalSIWEClient({ siweConfig }).client` loos off, we should make it so that `new Web3ModalSIWEClient({ siweConfig })` is the client. Config here should be for options.